helm-charts/sep-service/values.yaml

```diff
@@ -0,0 +1,134 @@
+fullName: anchor-platform
+namespace: sandbox
+service:
+   containerPort: 8080
+   servicePort: 8080
+   replicas: 1
+   type: NodePort
+   name: sep
+image:
+   repo: stellar
+   name: anchor-platform
+   tag: 022afab
+   pullPolicy: Always
+ingress:
+   labels:
+     app: appy
+     test: label_test
+   annotations:
+     kubernetes.io/ingress.class: "public"
+     cert-manager.io/cluster-issuer: "default"
+     ingress.kubernetes.io/ssl-redirect: "true"
+     ingress.kubernetes.io/browser-xss-filter: "true"
+     ingress.kubernetes.io/frame-deny: "true"
+     ingress.kubernetes.io/content-type-nosniff: "true"
+     ingress.kubernetes.io/hsts-max-age: "31536000"
+     ingress.kubernetes.io/hsts-include-subdomains: "true"
+   tls:
+     host: "www.stellaranchordemo.com"
+     secretName: anchor-platform-sep-server-cert
+   rules:
+      hosts:
+        - host: "{{ .Values.ingress.tls.host }}"
+          path: /
+          pathType: Prefix
+          backend:
+            servicePort: "{{ .Values.service.containerPort }}"
+            serviceName: "{{ .Values.fullName }}-svc-{{ .Values.service.name }}"
+stellar:
+   anchor:
+      config: in-memory
+      app-config:
+         type: config-spring-property                # Activate [config-spring-property] module
+         settings: "app-config"                        # The location of the configuration data
+      data-access:
+         type: data-spring-jdbc                      # Activate [config-spring-jdbc] module.
+         settings: data-spring-jdbc-sqlite           # The location of the configuration data in this file.
+      logging:
+         type: logging-logback
+         settings: logging-logback-settings
+   app_config:
+      app:
+         stellarNetworkPassphrase: Test SDF Network ; September 2015
+         hostUrl: https://www.stellaranchordemo.com
+         languages: en
+         horizonUrl: https://horizon-testnet.stellar.org
+         assets: assets-test.json
+         jwtSecretKey: secret
+      sep1:
+         enabled: true
+         stellarFile: file:/config/stellar-wks.toml
```

Comment:
Are users mean to be configuring this parameter at all? I suspect not and if so then maybe remove it completely from the values file to make it shorter?

Reply:
it is configurable now but with default value.